scripts/install.sh

```diff
@@ -18,14 +18,14 @@ function usage(){
     echo ""
     echo "Required Parameters:"
     echo ""
-    echo "    --issuerEndpoint (-i): This is the endpoint that mints the access_tokens and will also be the issuer in the access_token as well."
-    echo "    --oAuth2ApiEndpoint (-o): this is probably similar to your issuer endpoint but is the prefix to all OAuth2 APIs"
     echo "    --patientPickerEndpoint (-p): SMART on FHIR supports launch contexts and that will typically include a patient picker application that will proxy the /token and /authorize requests."
```

Comment:
Linden, I think we need to make the patientPickerEndpoint optional as well, and use the issuerEndpoint fetched from parameter store when the option is not provided.

Reply:
The patient picker URL is https://uxuj41rmw1.execute-api.us-west-2.amazonaws.com/dev which is created by okta-smartfhir-demo

the issuer endpoint is https://id-sbx.resmed.com/oauth2/aus115doo99N6zKxt0h8 which is created by okta

Reply:
yes, I have made the patientPickerEndpoint  optional now. The value would be `issuerEndpoint/v1` when patientPickerEndpoint  is not provided.